docs/Developer-Guide.md

```diff
@@ -482,7 +486,49 @@ See [Set up a debug console](#set-up-a-debug-console).
 $ sudo docker info 2>/dev/null | grep -i "default runtime" | cut -d: -f2- | grep -q runc  && echo "SUCCESS" || echo "ERROR: Incorrect default Docker runtime"
 ```
 
-## Set up a debug console
+## Set up a debug console(The easy way)
```

Comment:
Rather than putting the details in brackets, could you just change the structure of the MD file to something like the following as that will be clearer when viewed in the table of contents:

```md
## Set up a debug console

    :

### Simple debug console setup

    :

### Traditional debug console setup
```